File: snippets/run_actor.py
# Justdial Business Leads Scraper — https://apify.com/factden/justdial-business-leads
from apify_client import ApifyClient

client = ApifyClient("YOUR_APIFY_TOKEN")
run = client.actor("factden/justdial-business-leads").call(run_input={
    "cities": ["Mumbai"],
    "categories": ["Restaurants"],
    "maxItems": 100,
    "enrichContacts": True,
})
for lead in client.dataset(run["defaultDatasetId"]).iterate_items():
    print(lead["name"], lead["phone"], lead.get("email"), lead.get("gstNumber"))

File: snippets/run_actor.js
// Justdial Business Leads Scraper — https://apify.com/factden/justdial-business-leads
import { ApifyClient } from 'apify-client';

const client = new ApifyClient({ token: 'YOUR_APIFY_TOKEN' });
const run = await client.actor('factden/justdial-business-leads').call({
  cities: ['Mumbai'],
  categories: ['Restaurants'],
  maxItems: 100,
  enrichContacts: true,
});
const { items } = await client.dataset(run.defaultDatasetId).listItems();
items.forEach((l) => console.log(l.name, l.phone, l.email, l.gstNumber));
